src/acl_context.cpp

```diff
@@ -1202,7 +1202,7 @@ void acl_idle_update(cl_context context) {
   // firstly update the current context
   acl_update_context(context);
   // update the other contexts from the platform
-  for (auto _context : acl_platform.contexts_set) {
+  for (auto &_context : acl_platform.contexts_set) {
```

Comment:
Here's the trace:
```
src/acl_context.cpp:1205:13:
  Type: AUTO_CAUSES_COPY (AUTO_CAUSES_COPY)

src/acl_context.cpp:1205:13:
  1. auto_causes_copy: Using the "auto" keyword without an "&" causes the copy of an object of type _cl_context.
src/acl_context.cpp:1205:13:
  2. remediation: Use "auto &" instead of "auto".
```
Coverity seems to complain because of the copy duplicates of the pointer...

Reply:
Coverity does accept `const cl_context _context`. I think it just doesn't like `auto` copies, but allow other copies.